my_test.py

```diff
@@ -0,0 +1,13 @@
+"""This script tests SAC discrete"""
```

Comment:
Awesome PR, thanks for adding this variant to our SAC. This is a big value-add :)

Could we move this test script here into a new `tuned_example/sac/mountaincar_sac.py`?

See the already existing one for Pendulum, just copy that and change as needed.
Also, then add the new tuned example to our CI-testing suite by adding the entry into the BUILD file at the root of the `rllib/` dir. See the already existing entry for `pendulum_sac` and create a new entry one before it (they should be alphabetically sorted).